dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2175,14 +2163,6 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
-  @Deprecated
-  public static final PropertyKey MASTER_MOUNT_TABLE_ROOT_SHARED =
```

Comment:
most likely we still need this for some time?

Reply:
I didn't find where to use this one

Reply:
The "mount point" and "root mount" concepts may linger for a bit more. But I think this property key can be removed.